src/uu/ls/src/ls.rs

```diff
@@ -2420,12 +2421,31 @@ fn display_dir_entry_size(
     }
 }
 
-fn pad_left(string: &str, count: usize) -> String {
-    format!("{string:>count$}")
+trait ExtendPad {
```

Comment:
please a comment explaining why we do it

Reply:
Done.